### DISCUSSION TITLE
Support yyyy-MM-dd HH:mm:ss in events result list

### DISCUSSION THREAD

**Discussion by larenelg** [datalust/seq-tickets#1088] [Ideas]
Currently, we show event timestamp on the left in `dd MMM yyyy HH:mm:ss.fff`

Please let us know if being able to toggle to `yyyy-MM-dd HH:mm:ss` format would be useful, and why?

**Comment by andymac4182** [comment]
This would be great since it matches what I get from other systems logs that are text based. Also it is the same no matter where the system is built.

**Comment by Sebbl22** [comment]
+1 because it's the format I'm used to work with

**Comment by CrossBound** [comment]
I would like to see this because of personal preference (easier to read).

**Comment by colinangusmackay** [comment]
Allowing the user to generally specify their own date format would be ideal.

**Comment by nblumhardt** [comment]
![image](https://user-images.githubusercontent.com/342712/111416292-59847e00-872f-11eb-9023-8e28dd29e326.png)

This is shipping in 2021.2 🎉 

We're still open to supporting custom date formats, but will need to consider it down the track. Glad we could move this forward!

**Comment by JanEggers** [comment]
I have a related issue when formulating a query that includes @timestamp there is only the ToIsoString function. I was not able to fromat the date the same way it is rendered in the default view aka in local time

**Comment by JanEggers** [comment]
I have a related issue when formulating a query that includes @timestamp there is only the ToIsoString function. I was not able to fromat the date the same way it is rendered in the default view aka in local time
